build.sbt

```diff
@@ -690,27 +696,64 @@ lazy val `logging-service` = project
     frgaalJavaCompilerSetting,
     version := "0.1",
     libraryDependencies ++= Seq(
-      "org.slf4j"                   % "slf4j-api"     % slf4jVersion,
-      "com.typesafe"                % "config"        % typesafeConfigVersion,
-      "com.typesafe.scala-logging" %% "scala-logging" % scalaLoggingVersion,
-      akkaStream,
-      akkaHttp,
-      "io.circe"               %% "circe-core"      % circeVersion,
-      "io.circe"               %% "circe-parser"    % circeVersion,
-      "junit"                   % "junit"           % junitVersion              % Test,
-      "com.github.sbt"          % "junit-interface" % junitIfVersion            % Test,
-      "org.scalatest"          %% "scalatest"       % scalatestVersion          % Test,
-      "org.graalvm.nativeimage" % "svm"             % graalMavenPackagesVersion % "provided"
+      "org.slf4j"      % "slf4j-api" % slf4jVersion,
+      "com.typesafe"   % "config"    % typesafeConfigVersion,
+      "org.scalatest" %% "scalatest" % scalatestVersion % Test,
+      akkaHttp
     )
   )
+  .dependsOn(`logging-utils`)
+  .dependsOn(`logging-config`)
+
+lazy val `logging-jutil` = project
+  .in(file("lib/scala/logging-jutil"))
+  .configs(Test)
   .settings(
-    if (Platform.isWindows)
-      (Compile / unmanagedSourceDirectories) += (Compile / sourceDirectory).value / "java-windows"
-    else
-      (Compile / unmanagedSourceDirectories) += (Compile / sourceDirectory).value / "java-unix"
+    frgaalJavaCompilerSetting,
+    version := "0.1",
+    libraryDependencies ++= Seq(
+      "org.slf4j" % "slf4j-api" % slf4jVersion
+    )
+  )
+
+lazy val `logging-config` = project
+  .in(file("lib/scala/logging-config"))
+  .configs(Test)
+  .settings(
+    frgaalJavaCompilerSetting,
+    version := "0.1",
+    libraryDependencies ++= Seq(
+      "com.typesafe" % "config"    % typesafeConfigVersion,
+      "org.slf4j"    % "slf4j-api" % slf4jVersion
+    )
+  )
+
+lazy val `logging-logback` = project
+  .in(file("lib/scala/logging-logback"))
+  .configs(Test)
+  .settings(
+    frgaalJavaCompilerSetting,
+    version := "0.1",
+    libraryDependencies ++= Seq(
+      "org.slf4j"        % "slf4j-api"               % slf4jVersion,
+      "io.sentry"        % "sentry-logback"          % "6.28.0"           % Provided,
+      "org.netbeans.api" % "org-openide-util-lookup" % netbeansApiVersion % "provided"
```

Comment:
It seems this dependency can be removed

Reply:
This is needed for `LogbackSetup` and `LogbackLoggingServiceFactory` service registration